app/views/comments/thread.html.erb

```diff
@@ -1,7 +1,14 @@
 <% pingable = get_pingable(@comment_thread) %>
 
+<% if @post.parent.present? %>
+  <details>
+    <summary>Question</summary>
+    <%= render 'posts/expanded', post: @post.parent %>
+  </details>
+<% end %>
+
 <details>
-  <summary>Post</summary>
+  <summary> <% if !@post.parent.present? %> Thread on question <% else %> Thread on answer <% end %> </summary>
```

Comment:
This sort of indicates that the content behind the dropdown is the thread, rather than a post. I'd suggest leaving the wording as just "Post". With the change I suggested above that should be clear enough.